streams/streams-scala/src/main/scala/org/apache/kafka/streams/scala/kstream/KGroupedStream.scala

```diff
@@ -19,19 +19,8 @@ package kstream
 
 import org.apache.kafka.streams.kstream.internals.KTableImpl
 import org.apache.kafka.streams.scala.serialization.Serdes
-import org.apache.kafka.streams.kstream.{
-  SessionWindows,
-  Window,
-  Windows,
-  KGroupedStream => KGroupedStreamJ,
-  KTable => KTableJ
-}
-import org.apache.kafka.streams.scala.FunctionsCompatConversions.{
-  AggregatorFromFunction,
-  InitializerFromFunction,
-  ReducerFromFunction,
-  ValueMapperFromFunction
-}
+import org.apache.kafka.streams.kstream.{SessionWindows, SlidingWindows, Window, Windows, KGroupedStream => KGroupedStreamJ, KTable => KTableJ}
```

Comment:
nit: we want to keep this import and the one below in the same format as before, one import per line, to keep checkstyle happy